src/Symfony/Component/Notifier/Bridge/FakeChat/FakeChatEmailTransport.php

```diff
@@ -27,6 +27,8 @@
  */
 final class FakeChatEmailTransport extends AbstractTransport
 {
+    protected const HOST = 'default';
```

Comment:
```suggestion
    protected const DEFAULT_HOST = 'default';
```

Reply:
👎 this is defined in `AbstractTransport` and should be overridden